samples/client/petstore/kotlin-multiplatform/src/commonMain/kotlin/org/openapitools/client/infrastructure/ApiClient.kt

```diff
@@ -53,23 +44,9 @@ open class ApiClient(
     }
 
     companion object {
+        const val BASE_URL = "http://petstore.swagger.io/v2"
+        const val JSON_DEFAULT = Json { ignoreUnknownKeys = true }
```

Comment:
This forbidden `const` got also in the sample. I thought CI tries to at least build the sample projects?

Reply:
Maybe not the multiplaform sample projects, but the jvm kotlin client samples are build in CI.